runhouse/resources/hardware/launcher_utils.py

```diff
@@ -107,7 +107,15 @@ def supported_providers():
 
     @classmethod
     def sky_secret(cls):
+        from tests.fixtures.resource_fixtures import create_folder_path
+
         secrets_name = "ssh-sky-key"
```

Comment:
instead of changing logic to accomodate the test, we can save "ssh-sky-key" as a constant, and then just overwrite it in the test

`rh.constants.SSH_SKY_KEY = ""`

Reply:
yeah, fair point.